tests/unit/core/i18n/test_utils.js

```diff
@@ -509,4 +510,70 @@ describe(__filename, () => {
       expect(i18n.moment('1988-09-22').format('ll')).toEqual('22 sept. 1988');
     });
   });
+
+  describe('formatFilesize', () => {
+    const _formatFilesize = ({
+      _filesize,
+      _log,
+      i18n = fakeI18n(),
+      size = 123,
+    }) => {
+      return utils.formatFilesize({ _filesize, _log, i18n, size });
+    };
+
+    it('receives an expected string from filesize', () => {
```

Comment:
I don't think you need this test because `formats the number returned by filesize` covers the same thing.